### PR TITLE
[devtools] update WNDT63 multiclient

### PR DESCRIPTION
**Pull request by paulirish**
Just adding a bit more to be super clear.

### CODE REVIEW

File: src/content/en/updates/2017/10/devtools-release-notes.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Note: You can check what version of Chrome you're running at
 `chrome://version`. Chrome auto-updates to a new major version about every 6
 weeks.
 
-## Multi-client remote debugging support {: #multi-client }
+## Multi-client debugging support {: #multi-client }
```
Comment (kaycebasques) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: We described it as "Multi-client remote debugging support" in the UI

https://chromium-review.googlesource.com/c/chromium/src/+/713460/6/third_party/WebKit/Source/devtools/front_end/help/ReleaseNoteText.js#19

File: src/content/en/updates/2017/10/devtools-release-notes.md [hunk 2]
```diff
@@ -53,14 +53,15 @@ of VS Code and DevTools in action, side-by-side.
...
+on crbug.com][Crbug], and number 3 across the entire Chromium project.
 Multi-client support also  opens up quite a few interesting opportunities
 for integrating other tools with DevTools, or using those tools in new
 ways. For example:
 
 [Crbug]: https://crbug.com/129539
 
-* Two WebSocket protocol clients, such as two Puppeteer sessions, can now
+* An open DevTools window plus a connected protocol client. This includes Selenium/Chromedriver, Chrome debugging within VS Code or WebStorm, and WebSocket clients like Puppeteer. 
+* Two separate WebSocket protocol clients (eg. [Puppeteer](https://github.com/GoogleChrome/puppeteer) or [chrome-remote-interface](https://github.com/cyrus-and/chrome-remote-interface)), can now
```
Comment (kaycebasques) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: Writer nitpick: Can you change this to:

`Two separate WebSocket protocol clients, such as [Puppeteer](...) or [chrome-remote-interface](...), can now...`

Using something like "such as" rather than "e.g." comes from our style guide: https://developers.google.com/style/abbreviations#dont-use

File: src/content/en/updates/2017/10/devtools-release-notes.md (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: We described it as "Multi-client remote debugging" in the UI: https://chromium-review.googlesource.com/c/chromium/src/+/713460/6/third_party/WebKit/Source/devtools/front_end/help/ReleaseNoteText.js#19

File: src/content/en/updates/2017/10/devtools-release-notes.md (hunk 2 above)
Comment (kaycebasques) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: 1. Please break at 100 chars.
2. Can you rewrite to `Two separate WebSocket protocol clients, such as [Puppeteer](...) or [chrome-remote-interface](...), can now...`. Our style guide instructs us to [avoid latin abbreviations](https://developers.google.com/style/abbreviations#dont-use)

File: src/content/en/updates/2017/10/devtools-release-notes.md [hunk 3]
```diff
@@ -53,14 +53,15 @@ of VS Code and DevTools in action, side-by-side.
...
-on Crbug][Crbug], and number 3 across the entire Chromium project.
+on crbug.com][Crbug], and number 3 across the entire Chromium project.
 Multi-client support also  opens up quite a few interesting opportunities
 for integrating other tools with DevTools, or using those tools in new
 ways. For example:
 
 [Crbug]: https://crbug.com/129539
 
-* Two WebSocket protocol clients, such as two Puppeteer sessions, can now
+* An open DevTools window plus a connected protocol client. This includes Selenium/Chromedriver, Chrome debugging within VS Code or WebStorm, and WebSocket clients like Puppeteer.
```
Comment (kaycebasques) [inline src/content/en/updates/2017/10/devtools-release-notes.md]: Writer nitpick: Can you rephrase this to something like:

`Protocol clients such as ChromeDriver or the Chrome debugging extensions for VS Code and Webstorm, and WebSocket clients such as Puppeteer, can now run at the same time as DevTools.`

The rest of the items in the list are complete sentences, so this one should be, too. See [parallel construction](http://www.bartleby.com/141/strunk5.html#15)

**Comment by kaycebasques** [comment]
Thanks @paulirish 

You can ignore these warnings:

src/content/en/updates/2017/10/devtools-release-notes.md - No wf_blink_components field found in metadata. Add if appropriate.

src/content/en/updates/2017/10/devtools-release-notes.md#L181 - Updates post is missing comment widget: {% include "comment-widget.html" %}

**Comment by paulirish** [comment]
updated
